source/Dialog.cpp

```diff
@@ -382,7 +382,8 @@ void Dialog::Init(const string &message, Truncate truncate, bool canCancel, bool
 	// Resize textRectSize to match the visual height of the dialog, which will
 	// be rounded up from the actual text height by the number of panels that
 	// were added. This helps correctly position the TextArea scroll buttons.
-	textRectSize.Y() = 60 + height * 40 - 30 * (!isMission && (intFun || stringFun));
+	// The text height was over-reported by 6 pixels, so we add those pixels back for consistency.
+	textRectSize.Y() = 60 + height * 40 - 30 * (!isMission && (intFun || stringFun)) + 6;
```

Comment:
```suggestion
	textRectSize.Y() = 60 + height * 40 + 6 - 30 * (!isMission && (intFun || stringFun));
```
I think having it at the end makes it too easy to miss it, especially because the 30 is only subtracted some of time time. I think this also more closely matches the order of the other line, which this is sort of inversing.

It would probably be better if all of these constants were named constants, so we don't need to try to reverse engineer what each value represents, but that's out off scope for fixing this bug.

Reply:
Yes, I'm planning to extract these values to static fields in Dialog, but that's for another PR.